client/src/app/dashboard/components/account-card/account-card.component.ts

```diff
@@ -1,18 +1,32 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { User } from 'src/app/user';
+
+import { ReadAccountComponent } from '../../dialogs/read-account/read-account.component';
+import { Account } from '../../models';
 
 @Component({
   selector: 'app-account-card',
   templateUrl: './account-card.component.html',
   styleUrls: ['./account-card.component.scss'],
 })
 export class AccountCardComponent {
+  @Input() id: string = '';
   @Input() name: string = '';
   @Input() balance: string = '';
   @Input() symbol: string = '';
 
   @Input() selected: boolean = false;
   hoveredOn: boolean = false;
 
+  @Output() clickEvent = new EventEmitter<Account>();
+
+  @Output() editAccountEvent = new EventEmitter<Account>();
```

Comment:
I think it's not the best idea to make `Input`/`Output` for every action/entity/property. It makes component clumsy. You can combine events/data, or inject API services and retrieve data by ids here.